index.js

```diff
@@ -2,6 +2,10 @@ function isVinnikCool() {
   return true;
 }
 
+function Nino() {
```

Comment:
Add function to export, to bring this amazing function to the world

Reply:
Nice catch, added